components/jupyter-web-app/kubeflow_jupyter/default/templates/notebooks.html

```diff
@@ -1,80 +1,99 @@
 {% extends "layout.html" %}
 {% block javascript %}
-  {{ super() }}
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.11/lodash.min.js"></script>
-  <script src="{{ prefix + url_for('static', filename='js/notebooks.js') }}"></script>
+	{{ super() }}
+    <script defer>
+        // Establish communication with Central Dashboard
+        window.addEventListener('DOMContentLoaded', function (event) {
+            const CD_EvHandler = (window.centraldashboard || {}).CentralDashboardEventHandler;
```

Comment:
*Nit:* `const` is still an ES6 feature if we're worried about behavior in browsers without native support.